### PR TITLE
Fixing commands to register miner and validator and general cleanup

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Critical changes to the docs in this PR are:
1. Registration steps for staging, testnet and mainnet docs. 
2. The step "Editing the default arguments". See the Discord thread: https://discord.com/channels/@me/1164614647614279811/1164705982165827646.
3. Rest are cleanup and formatting changes.

### CODE REVIEW

File: docs/running_on_staging.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
-This tutorial will guide you through setting up a local subtensor chain, creating a subnetwork, and connecting your mechanism to it.
+This tutorial will guide you through setting up a local blockchain (subtensor), creating a subnet, and run your incentive mechanism on the subnet.
 
-### 1. Install substrate dependencies
+## Prerequisites
+
+Before proceeding further, make sure that you have installed Bittensor and the Wallet. See the below instructions:
+
+- [Install `bittensor`](https://github.com/opentensor/bittensor#install).
+- [Install `wallet`](https://github.com/opentensor/bittensor#wallets).
```
Comment (ifrit98) [inline docs/running_on_staging.md:10]: Install -> Create wallet.

We used to have the wallet separated out into its own package, so this language could cause some confusion.

File: docs/running_on_testnet.md [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
+- Only use your testnet wallet.
+- Do not reuse the password of your mainnet wallet.
+- Make sure your incentive mechanism is resistant to abuse. 
+
+## Prerequisites
+
+Before proceeding further, make sure that you have installed Bittensor and the Wallet. See the below instructions:
+
+- [Install `bittensor`](https://github.com/opentensor/bittensor#install).
+- [Install `wallet`](https://github.com/opentensor/bittensor#wallets).
```
Comment (ifrit98) [inline docs/running_on_testnet.md:18]: Install -> Create

**Review by ifrit98 — APPROVED** [review]
LGTM! Left you a single nit.

**Comment by rajkaramchedu** [comment]
I will merge this and take up the item 2 (in the description) in a separate PR.

**Pull request merged**